templates/layout/parts/objectlock_message.html.twig

```diff
@@ -0,0 +1,151 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2024 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+#}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% import 'components/form/basic_inputs_macros.html.twig' as inputs %}
+
+{% macro forceUnlockButton(item, check_unlock_right) %}
+    {% set real_profile = session('glpilocksavedprofile')|default(null) %}
+    {% if not check_unlock_right or (real_profile != null and real_profile[item.fields['itemtype']] b-and constant('UNLOCK')) %}
+        <button type="button" class="btn btn-sm btn-primary ms-2 force-unlock-item">
+            <i class="ti ti-lock-open"></i>
+            <span>{{ __('Force unlock %1s #%2s')|format(item.fields['itemtype']|itemtype_name, item.fields['items_id']) }}</span>
+        </button>
+        <script>
```

Comment:
Translations present in a `<script>` tag inside a Twig template are not detected by the current `bin/console/extract-template` script. I am not sure it would be easy to change this.

Anyway, it would be preferable to move this script into the new `ObjectLock` module as much as possible. It would permit to correctly extranct its translation and also to make the JS lint job pass on it.